README.md

```diff
@@ -238,6 +238,12 @@ and scalable image contents such as SVG and Lottie Animation among the Tizen pow
   <img width="798" height="285" src="https://github.com/Samsung/thorvg/blob/master/res/example_tizen.png">
 </p>
 
+[Back to contents](#contents)
+<br />
+<br />
+#### Tizen NUI
+[Natural User Interface (NUI)](https://docs.tizen.org/application/dotnet/guides/user-interface/nui/overview/) is one of the ui framework provided by Tizen, which is written in C#. ThorVG is a backend engine for the [Vector Graphics](https://docs.tizen.org/application/dotnet/guides/user-interface/nui/vectorgraphics/Overview/) API set provided by the NUI. The NUI's VectorGraphics APIs are almost 1: 1 with the APIs of ThorVG. This is a good example that ThorVG is used by the ui framework level
```

Comment:
I'd like to suggest to integrate the content in "Practices/Tizen" rather than separating a new section "Tizen NUI"

Reply:
Or you can leave it to me.

Reply:
I'll leave it to you. This PR will be closed.